resources/translations/fi.edn

```diff
@@ -146,7 +147,15 @@
                           :submitted "Haettu"
                           :unknown "Tuntematon"}
                  :submitted "Lähetetty"
-                 :latest-activity "Viimeisin toimenpide"
+                 :todo "Toimia tarvitaan"
+                 :todos {:new-application "Uusi hakemus"
+                         :no-pending-requests "Kaikkiin pyyntöihin on vastattu"
+                         :resubmitted-application "Uudelleenlähetetty hakemus"
+                         :unknown "Tuntematon"
```

Comment:
"Tuntematon" sounds bad. Is it shown in any real case currently?

Reply:
The same as with event types and states. If the backend returns some new value, it'll use that unknown.